backend/src/main/java/org/eclipse/tractusx/puris/backend/masterdata/domain/model/MaterialPartnerRelation.java

```diff
@@ -47,8 +47,12 @@ public class MaterialPartnerRelation {
     @EmbeddedId
     private Key key;
 
+    @Pattern(regexp = PatternStore.URN_STRING)
```

Comment:
I would rename the pattern to include UUID, as the URN is only one of the valid formats of the uuid accepted here

Reply:
Done